gui/src/components/tracker/TrackersTable.tsx

```diff
@@ -290,9 +290,9 @@ export function TrackersTable({
           (device?.hardwareStatus?.rssi != null ||
             device?.hardwareStatus?.ping != null) && (
```

Comment:
if this check is here.
None of the ping !=null and rssi != null should matter?

Reply:
my bad cant read. But i dont think this condition is needed anymore

Reply:
The wifi section is disabled if neither rssi nor ping exist, which covers the case when a tracker has just connected but sent no statuses. This is the same as battery, and is also the original behavior, so I did not need to change it